File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        res=defaultdict(list)
        for s in strs:
            key=[0]*26
            for a in s:
                key[ord(a)-ord('a')]+=1
            res[tuple(key)].append(s)
        return list(res.values())

File: Data Structures & Algorithms/time-needed-to-buy-tickets/submission-0.java
class Solution {
    public int timeRequiredToBuy(int[] tickets, int k) {
        int res=0;
        while(true){
            for(int i=0;i<tickets.length;i++){
                if(tickets[i]>0){
                    tickets[i]--;
                    res++;
                    if(i==k && tickets[k]==0) return res;
                }
                
            }
        }
    }
}